py/gc.c

```diff
@@ -701,6 +701,11 @@ void gc_info(gc_info_t *info) {
 
     info->used *= BYTES_PER_BLOCK;
     info->free *= BYTES_PER_BLOCK;
+
+    #if MICROPY_GC_SPLIT_HEAP_AUTO
+    info->max_new_split = gc_get_max_new_split();
```

Comment:
This line had a missing semicolon!  I worked only because the GC_EXIT macro below is empyt when auto split heap is enabled...

Reply:
Oops, thanks @dpgeorge!